app/src/main/res/layout/view_fireproof_website_settings_selection.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--<androidx.appcompat.widget.SwitchCompat xmlns:android="http://schemas.android.com/apk/res/android"-->
+<!--    android:id="@+id/fireproofWebsiteToggle"-->
+<!--    style="@style/FireproofWebsiteSwitch"-->
+<!--    android:paddingStart="16dp"-->
+<!--    android:paddingEnd="16dp"-->
+<!--    android:text="@string/fireproofWebsiteToogleText"-->
+<!--    android:theme="@style/SettingsSwitchTheme" />-->
+
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/fireproofWebsiteSettingsSelection"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:padding="16dp">
```

Comment:
Could we use `keylines` here rather than hardcoded dp values?
```suggestion
        android:padding="@dimen/keyline_4">
```

Reply:
Sure, I will use them from now on 👍